README.md

```diff
@@ -142,8 +175,9 @@ No-op.
 # The put step
 
 Sets the GitHub commit status for a given commit, following the [GitHub Commit status API].
+The same commit can have multiple statuses, differentiated by parameter `context`.
```

Comment:
but you don't see this in the Github? commit is either pass or fail? except of course for build status?

Reply:
I am not sure I understand the question :-)
This is just a description of the GitHub API. You can see them in the screenshot that we already have in the README.

Reply:
See also for example https://github.com/Pix4D/cogito-test-read-write/commits/stable and click on any X or V sign to see.